pages/directory/_regions/_departements/_communes/_categories/index.vue

```diff
@@ -0,0 +1,42 @@
+<script>
+import DirectoryList from "~/components/directory_list";
+
+export default {
+  extends: DirectoryList,
+  data() {
+    return {
+      propertyLabel: [
+        { key: "name" },
+        { key: "cat", translation: "categories." }
+      ],
+      itemKey: "pois",
+      displayIcon: true
+    };
+  },
+  methods: {
+    itemLink(item) {
+      return "/place/" + item.id;
+    }
+  },
+  asyncData({ params, query }) {
+    return DirectoryList.fetchData({
+      region: params.regions,
+      departement: params.departements,
+      commune: params.communes,
```

Comment:
Am I right to assume that these refer to `boundary=administrative;admin_level=4|5|8`?

In Germany, we have `admin_level=5` (Regierungsbezirk / state-district) only in certain federal states, but instead `admin_level=6` (Landkreis / county) is quite relevant. This seems to be the same for Austria and Switzerland with Austria not having any `admin_level=5`.

(Source: https://wiki.openstreetmap.org/wiki/Tag:boundary%3Dadministrative#11_admin_level_values_for_specific_countries)
In Switzerland, it seems similar.

Reply:
> 
> Am I right to assume that these refer to `boundary=administrative;admin_level=4|5|8`?
> 
Levels for France are (countrywide) :
4 : region
6 : departement
8 : commune

Reply:
Oh, sorry, my mistake.